src/main/java/com/uid2/shared/model/Site.java

```diff
@@ -123,6 +157,7 @@ public String toString() {
                 ", description=" + description +
                 ", enabled=" + enabled +
                 ", domain_names=" + domainNames.toString() +
+                ", app_names=" + domainNames.toString() +
```

Comment:
Should be `appNames.toString()`

Reply:
@caroline-ttd we're using Lombok elsewhere in shared so you could consider using that instead

Reply:
could you look at adding a unit test that could have caught this bug if it's not too complicated?

Reply:
I didn't find a good example of Lombok in our Shared repo, its mainly enums using Lombok at this point. Using Lombok's `@Data` should be enough to replace the getters, setters, ToString and equals method. The `hashCode` method looks different than Lombok's implementation and I am not sure where we are using this

Reply:
@asloobq I don't think we're actually using these methods outside of test code. I've been asking that we stop adding overrides to new classes unless there's a good reason for them.

Reply:
I added a new simple test for toString() and use `@Data` to use setters and getters. I keep the `toString()` and `hashCode()` as it is different than Lombok's implementation.

Reply:
Different in what way? Maybe the Lombok implementation is better?
The idea is to prevent bugs due to missing a field.

Reply:
In our code, the` toString()` method presents data in the format `Site{id=1, name='NAME', ...}`, while in Lombok, it's displayed as `Site(id=1, name='NAME', ...)`. These representations differ in two aspects: 1. The original format employs single quotes (') around the name field, which is absent in the Lombok version. 2. The original format utilizes curly braces `{}`, whereas Lombok uses parentheses `()`.

Reply:
Sure, we don't need to retain the existing behavior though. This is just used in logging right (if it's used at all)?

Reply:
If the `toString() `output format is pre-defined, I will remove this method and let Lombok handles it.